### PR TITLE
Modify only username in test session

### PR DESCRIPTION
**Pull request by lukasz-walkiewicz**
## Release notes

(x) This is not user-visible or docs only and no release notes are required.
( ) Release notes are required, please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

**Review by findepi — APPROVED** [review]
Failure looks related

```
Error:  io.trino.plugin.hive.TestHiveConnectorSmokeTest.testCreateSchemaWithNonLowercaseOwnerName  Time elapsed: 0.095 s  <<< FAILURE!
java.lang.AssertionError: 

Expecting throwable message:
  "Access Denied: Cannot set role admin"
to contain:
  "Access Denied: Cannot create schema"
but did not.

```

**Comment by findepi** [comment]
@ebyhr @krvikash it seems `testCreateSchemaWithNonLowercaseOwnerName` was recently added.
can you please suggest how the Hive version should be updated?

File: testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorSmokeTest.java [hunk 1]
```diff
@@ -322,8 +323,11 @@ public void testCreateSchemaWithNonLowercaseOwnerName()
     {
         skipTestUnless(hasBehavior(SUPPORTS_CREATE_SCHEMA));
 
-        Session newSession = Session.builder(getSession())
-                .setIdentity(Identity.ofUser("ADMIN"))
+        Session session = getSession();
```
Comment (ebyhr) [inline testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorSmokeTest.java:326]: The same test exists in `BaseConnectorTest`. Can we also update the test?

File: testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorSmokeTest.java [hunk 2]
```diff
@@ -322,8 +323,11 @@ public void testCreateSchemaWithNonLowercaseOwnerName()
...
+        Session newSession = Session.builder(session)
+                .setIdentity(Identity.from(session.getIdentity())
+                        .withUser(session.getUser().toUpperCase(ENGLISH))
+                        .build())
```
Comment (ebyhr) [inline testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorSmokeTest.java:330]: > can you please suggest how the Hive version should be updated?

Let's simply update the `hasMessageContaining` line.

**Review by ebyhr — APPROVED** [review]

**Comment by mosabua** [comment]
@lukasz-walkiewicz @ebyhr @findepi .. this looks super close to merge. Can you follow up

**Comment by mosabua** [comment]
Ping @lukasz-walkiewicz @ebyhr @findepi

**Pull request closed without merging**